File: core/src/frontendServer/server/models/MachineGunPickup.js
const Pickup = require('./Pickup');
const constants = require('../settings/constants');
const ProjectileType = require('./ProjectileType');

class MachineGunPickup extends Pickup {
    constructor(x, y) {
        super(x, y, constants.MACHINE_GUN_SPRITE_WIDTH, constants.MACHINE_GUN_SPRITE_HEIGHT, ProjectileType.MACHINE_GUN)
    }
}

module.exports = MachineGunPickup;

File: core/src/com/mygdx/game/model/MachineGunPickup.kt
package com.mygdx.game.model

import com.badlogic.gdx.graphics.Texture
import com.mygdx.game.settings.MACHINE_GUN_SPRITE_HEIGHT
import com.mygdx.game.settings.MACHINE_GUN_SPRITE_WIDTH

class MachineGunPickup(x: Float = 0f, y: Float = 0f, texture: Texture) : Pickup(x, y, MACHINE_GUN_SPRITE_WIDTH,
        MACHINE_GUN_SPRITE_HEIGHT, texture)